infra/src/main/kotlin/fi/fta/geoviite/infra/util/ResultSetExternal.kt

```diff
@@ -99,6 +101,15 @@ fun <T> ResultSet.getOid(name: String): Oid<T> = verifyNotNull(name, ::getOidOrN
 
 fun <T> ResultSet.getOidOrNull(name: String): Oid<T>? = getString(name)?.let(::Oid)
 
+fun <T> ResultSet.getRatkoExternalId(oidName: String, planItemIdName: String): RatkoExternalId<T> =
+    verifyNotNull(oidName, planItemIdName, ::getRatkoExternalIdOrNull)
+
+fun <T> ResultSet.getRatkoExternalIdOrNull(oidName: String, planItemIdName: String): RatkoExternalId<T>? {
+    val oid = getOidOrNull<T>(oidName)
+    val planItemId = getIntOrNull(planItemIdName)?.let(::RatkoPlanItemId)
+    return if (oid == null) null else RatkoExternalId(oid, planItemId)
```

Comment:
Tämäkin fine ja makuasiaa, mutta itse tykkään 1-return tyylistä silloin kun se on näin helppo:
 ```
return getOidOrNull<T>(oidName)?.let { oid ->
    RatkoExternalId(oid, getIntOrNull(planItemIdName)?.let(::RatkoPlanItemId))
}
```